h/http2-wrapper/http2-wrapper_v2.1.10_ubi8.5.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : http2-wrapper
+# Version       : v2.1.10
+# Source repo   : https://github.com/szmarczak/http2-wrapper.git 
+# Tested on     : UBI8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nailusha Potnuru <pnailush@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Exit immediately if a command exits with a non-zero status.
+PACKAGE_NAME=http2-wrapper
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-v2.1.10}
+PACKAGE_URL=https://github.com/szmarczak/http2-wrapper.git
+yum -y update && yum install -y npm  git gcc jq
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+                echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 0
+fi
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+PACKAGE_VERSION=$(jq -r ".version" package.json)
+# run the test command from test.sh
+if ! npm install && npm audit fix && npm audit fix --force; then
+        echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails" 
+        exit 1
+fi
+if ! npm test; then
+        echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails" 
+        exit 1
+else
+        echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" 
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success" 
+        exit 0
+fi
+#This script build pass and tests are in parity due to below mentioned error
```

Comment:
Travis build is failing with other error.

Reply:
Updated script